view.php

```diff
@@ -55,7 +55,9 @@
 $groupselector = '';
 $groupid = 0;
 
-if (!empty($cm->groupmode) && $group != 0) {
+$groupmode = groups_get_activity_groupmode($cm, $course);
+
+if (!empty($groupmode) && $group != 0) {
```

Comment:
As far as I read into the code of `groups_get_activity_groupmode` it will automatically handle the case that the "group" page param is not specified. So you probably can get rid of the definition of the param?

In which cases do we have "$group == 0"?